source/community/release_notes/redwood.rst

```diff
@@ -13,6 +13,47 @@ These are the release notes for the Redwood release, the 18th community release
 Breaking Changes
 ****************
 
+-  Deployers must ensure that their
+   ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']`` Django setting in LMS
+   contains the full complement of private key numbers.*
+
+   -  **Background:**\ *In LMS, we are switching from the pyjwkest
+      library to PyJWT for signing JWTs. (pyjwkest is now unmaintained.)
+      However, PyJWT has stricter requirements for the private key in
+      ``JWT_PRIVATE_SIGNING_JWK``. Before you upgrade to Redwood, you
+      will need to update this key using a script. Otherwise, JWT
+      signing will fail, and users will be unable to log in.*
```

Comment:
I don't think this section needs to be in italics. Maybe the rendering will work better without it. 

Also, I think this would read more clearly if it were in the past tense. e.g. "In LMS, we switched from the pyjwkest library..."

Reply:
Good catch. I used a Confluence -> RTF and then RTF to RST converter on this and I suspect that's an artifact.